src/main/java/seedu/address/storage/JsonAdaptedEvent.java

```diff
@@ -23,15 +24,19 @@ class JsonAdaptedEvent {
 
     private final String eventName;
 
+    private final String eventDateTime;
```

Comment:
should this be eventDate or is eventDateTime okay

Reply:
Fixed variable name to eventDate